fastlane/Fastfile

```diff
@@ -145,3 +143,13 @@ lane :sonarqube do |options|
     sonar_runner_args: runner_args
   )
 end
+
+desc "Updates the copyright year of license headers in source files if needed"
+lane :copyright do |options|
+  ensure_git_status_clean
+
+  sh("../scripts/update-license-year.sh")
+  sh("git diff-index --quiet HEAD || git commit -am 'Update license header year - happy new year'")
```

Comment:
This is embarrassing but I never knew you could combine -a and -m in git... that's gonna save me some time 🤦‍♂️